ibis/backends/flink/utils.py

```diff
@@ -326,3 +327,14 @@ def translate_literal(op: ops.Literal) -> str:
         return f"ARRAY{list(value)}"
 
     raise NotImplementedError(f"No translation rule for {dtype}")
+
+
+def ibis_schema_to_flink_schema(schema: sch.Schema) -> RowType:
```

Comment:
Not a change request, but a question (maybe one of the maintainers knows the answer best):

What is the "right" way to do this? For example, [other backends implement `SchemaMapper`](https://github.com/ibis-project/ibis/blob/96243416771109fbaf383167470ac2481fe938ef/ibis/backends/bigquery/datatypes.py#L101); is that the current recommended way, or is it right to do something simple like this?

Reply:
subclassing `SchemaMapper` is the way to go.

Reply:
Thanks for the suggestion. Implemented both SchemaMapper and TypeMapper now.